.murdock

```diff
@@ -207,7 +209,7 @@ get_supported_boards() {
     for board in $boards
     do
         echo $board
-    done | $(_greplist $BOARDS)
+    done | $(_greplist $BOARDS) | $(_greplist esp32-heltec-lora32-v2 esp8266-esp-12x)
```

Comment:
This is just temporary, to limit the boards that are being build for rapid iteration on the CI while making changes on the PR, it is a `REMOVEME:` commit (it blocks merging), I'll remove it once we are done.

Reply:
OK, I'm wondering whether it would make sense also to add `esp32-wrover-kit` because it is the one of the most featured board and I already figured out a missing dependency, see https://github.com/RIOT-OS/RIOT/pull/17232#discussion_r759054428.

Reply:
Sure, I'll add all of them

Reply:
All boards added in 78aff83565141d764a561f55505f5f32b0dda927